galasa-ui/public/static/markdown/home-contents.md

```diff
@@ -0,0 +1,6 @@
+# Welcome to your Galasa Service
+Get the most from your Galasa experience by reading the [Galasa documentation](https://galasa.dev/)
+This is a test
+<br />
```

Comment:
Do we need "This is a test"? Also there's a missing `.` after `Galasa documentation`.